test/Microsoft.AspNetCore.Routing.Tests/Internal/HttpMethodEndpointConstraintTest.cs

```diff
@@ -27,7 +27,7 @@ public class HttpMethodEndpointConstraintTest
 
         [Theory]
         [MemberData(nameof(AcceptCaseInsensitiveData))]
-        public void HttpMethodEndpointConstraint_IgnoresPreflightRequests(IEnumerable<string> httpMethods, string accessControlMethod)
+        public void HttpMethodEndpointConstraint_AcceptsPreflightRequests(IEnumerable<string> httpMethods, string accessControlMethod)
```

Comment:
We're also going to need the opposite test. An action that doesn't handle CORS requests should not allow a CORS request through